src/virtualship/cli/_plan.py

```diff
@@ -0,0 +1,1174 @@
+                ]
+
+            # save
+            self.schedule.to_yaml(f"{self.path}/schedule.yaml")
+            return True
+
+        except Exception as e:
+            log_exception_to_file(
+                e, self.path, context_message="Error saving ship config:"
+            )
+
+            raise UnexpectedError(
+                UNEXPECTED_MSG_ONSAVE
+                + f"\n\nTraceback will be logged in {self.path}/virtualship_error.txt. Please attach this/copy the contents to any issue submitted."
+            ) from None
+
+
+class ConfigEditor(Container):
+    DEFAULT_ADCP_CONFIG: ClassVar[dict[str, float]] = {
+        "num_bins": 40,
+        "period_minutes": 5.0,
+    }
+
+    DEFAULT_TS_CONFIG: ClassVar[dict[str, float]] = {"period_minutes": 5.0}
+
+    INSTRUMENT_FIELDS: ClassVar[dict[str, dict]] = {
+        "adcp_config": {
+            "class": ADCPConfig,
+            "title": "Onboard ADCP",
+            "attributes": [
+                {"name": "num_bins"},
+                {"name": "period", "minutes": True},
+            ],
+        },
+        "ship_underwater_st_config": {
+            "class": ShipUnderwaterSTConfig,
+            "title": "Onboard Temperature/Salinity",
+            "attributes": [
+                {"name": "period", "minutes": True},
+            ],
+        },
+        "ctd_config": {
+            "class": CTDConfig,
+            "title": "CTD",
+            "attributes": [
+                {"name": "max_depth_meter"},
+                {"name": "min_depth_meter"},
+                {"name": "stationkeeping_time", "minutes": True},
+            ],
+        },
+        "ctd_bgc_config": {
+            "class": CTD_BGCConfig,
+            "title": "CTD-BGC",
+            "attributes": [
+                {"name": "max_depth_meter"},
+                {"name": "min_depth_meter"},
+                {"name": "stationkeeping_time", "minutes": True},
+            ],
+        },
+        "xbt_config": {
+            "class": XBTConfig,
+            "title": "XBT",
+            "attributes": [
+                {"name": "min_depth_meter"},
+                {"name": "max_depth_meter"},
+                {"name": "fall_speed_meter_per_second"},
+                {"name": "deceleration_coefficient"},
+            ],
+        },
+        "argo_float_config": {
+            "class": ArgoFloatConfig,
+            "title": "Argo Float",
+            "attributes": [
+                {"name": "min_depth_meter"},
+                {"name": "max_depth_meter"},
+                {"name": "drift_depth_meter"},
+                {"name": "vertical_speed_meter_per_second"},
+                {"name": "cycle_days"},
+                {"name": "drift_days"},
+            ],
+        },
+        "drifter_config": {
+            "class": DrifterConfig,
+            "title": "Drifter",
+            "attributes": [
+                {"name": "depth_meter"},
+                {"name": "lifetime", "minutes": True},
```

Comment:
Make clear in the tool that lifetime is in minutes? And why not days, like Argo?